sequence_processing_pipeline/human-depletion.yaml

```diff
@@ -0,0 +1,103 @@
+  - h5py=3.9.0=nompi_py311he78b9b8_101
+  - hdf5=1.14.1=nompi_h4f84152_100
+  - hmmer=3.3.2=hdbdd923_4
+  - htslib=1.17=h81da01d_2
+  - isa-l=2.30.0=ha770c72_4
+  - k8=0.2.5=hdcf5f25_4
+  - kernel-headers_linux-64=2.6.32=he073ed8_16
+  - keyutils=1.6.1=h166bdaf_0
+  - krb5=1.21.1=h659d440_0
+  - ld_impl_linux-64=2.40=h41732ed_0
+  - libaec=1.0.6=hcb278e6_1
+  - libblas=3.9.0=17_linux64_openblas
+  - libcblas=3.9.0=17_linux64_openblas
+  - libcurl=8.2.1=hca28451_0
+  - libdeflate=1.18=h0b41bf4_0
+  - libedit=3.1.20191231=he28a2e2_2
+  - libev=4.33=h516909a_1
+  - libexpat=2.5.0=hcb278e6_1
+  - libffi=3.4.2=h7f98852_5
+  - libgcc-devel_linux-64=13.1.0=he3cc6c4_0
+  - libgcc-ng=13.1.0=he5830b7_0
+  - libgfortran-ng=13.1.0=h69a702a_0
+  - libgfortran5=13.1.0=h15d22d2_0
+  - libgomp=13.1.0=he5830b7_0
+  - libidn2=2.3.4=h166bdaf_0
+  - liblapack=3.9.0=17_linux64_openblas
+  - libnghttp2=1.52.0=h61bc06f_0
+  - libnsl=2.0.0=h7f98852_0
+  - libopenblas=0.3.23=pthreads_h80387f5_0
+  - libsanitizer=13.1.0=hfd8a6a1_0
+  - libsqlite=3.42.0=h2797004_0
+  - libssh2=1.11.0=h0841786_0
+  - libstdcxx-ng=13.1.0=hfd8a6a1_0
+  - libunistring=0.9.10=h7f98852_0
+  - libuuid=2.38.1=h0b41bf4_0
+  - libzlib=1.2.13=hd590300_5
+  - minigraph=0.20=he4a0461_2
+  - minimap2=2.26=he4a0461_1
+  - more-itertools=10.1.0=pyhd8ed1ab_0
+  - ncurses=6.4=hcb278e6_0
+  - numpy=1.25.2=py311h64a7726_0
+  - openssl=3.1.2=hd590300_0
+  - pcre=8.45=h9c3ff4c_0
+  - perl=5.32.1=4_hd590300_perl5
+  - perl-archive-tar=2.40=pl5321hdfd78af_0
+  - perl-carp=1.50=pl5321hd8ed1ab_0
+  - perl-common-sense=3.75=pl5321hd8ed1ab_0
+  - perl-compress-raw-bzip2=2.201=pl5321h166bdaf_0
+  - perl-compress-raw-zlib=2.202=pl5321h166bdaf_0
+  - perl-encode=3.19=pl5321h166bdaf_0
+  - perl-exporter=5.74=pl5321hd8ed1ab_0
+  - perl-exporter-tiny=1.002002=pl5321hd8ed1ab_0
+  - perl-extutils-makemaker=7.70=pl5321hd8ed1ab_0
+  - perl-io-compress=2.201=pl5321hdbdd923_2
+  - perl-io-zlib=1.14=pl5321hdfd78af_0
+  - perl-json=4.10=pl5321hdfd78af_0
+  - perl-json-xs=2.34=pl5321h4ac6f70_6
+  - perl-list-moreutils=0.430=pl5321hdfd78af_0
+  - perl-list-moreutils-xs=0.430=pl5321h031d066_2
+  - perl-parent=0.241=pl5321hd8ed1ab_0
+  - perl-pathtools=3.75=pl5321h166bdaf_0
+  - perl-scalar-list-utils=1.63=pl5321h166bdaf_0
+  - perl-storable=3.15=pl5321h166bdaf_0
+  - perl-types-serialiser=1.01=pl5321hdfd78af_0
+  - pip=23.2.1=pyhd8ed1ab_0
+  - python=3.11.4=hab00c5b_0_cpython
+  - python_abi=3.11=3_cp311
+  - readline=8.2=h8228510_1
+  - repeatmasker=4.1.5=pl5321hdfd78af_0
+  - rmblast=2.14.0=h4565617_2
+  - samtools=1.17=hd87286a_1
+  - seqtk=1.4=he4a0461_1
+  - setuptools=68.0.0=pyhd8ed1ab_0
+  - sysroot_linux-64=2.12=he073ed8_16
+  - tk=8.6.12=h27826a3_0
+  - trf=4.09.1=h031d066_4
+  - tzdata=2023c=h71feb2d_0
+  - wget=1.20.3=ha35d2d1_1
+  - wheel=0.41.1=pyhd8ed1ab_0
+  - xz=5.2.6=h166bdaf_0
+  - zlib=1.2.13=hd590300_5
+  - zstd=1.5.2=hfc55251_7
+prefix: /home/mcdonadt/miniconda3/envs/human-depletion
```

Comment:
This looks linked to @wasade home dir, is this file actually needed?

Reply:
No idea what this is or why it's here

Reply:
@charles-cowart, I meant the whole file; I don't think is needed, right?

Reply:
Okay. At the time I thought it would be good to include it for future reference, but yes, it isn't needed.